src/main/resources/interface-specification.yml

```diff
@@ -151,50 +151,50 @@ paths:
             application/problem+json:
               schema:
                 $ref: '#/components/schemas/Problem'
-  /organizations/{id}:
+  /institutions/{id}:
```

Comment:
is this still the internal id? it could be ambiguous also considering line 209.
Same applies for all endpoints `/institutions/{id}`

Reply:
I changed `/institutions/external/{id}` to `/institutions/external/{externalId}`, but I kept all `/institutions/{id} *` endpoints unchanged.